hdr/sqlite_modern_cpp.h

```diff
@@ -173,7 +174,12 @@ namespace sqlite {
       return sqlite3_sql(_stmt.get());
     }
 
-		void used(bool state) { execution_started = state; }
+		void used(bool state) {
+			if(execution_started == true && state == true) {
+				throw sqlite_exception("Already used statement executed again! Please reset() first!",sql());
```

Comment:
I was pondering that too, but the "special" exceptions are the SQLite errors so i did not want to add custom ones there. Because SQLite wouldnt throw an error there.